### PR TITLE
Fix Dockerfile to run models

### PR DESCRIPTION
**Pull request by CasperKjaerhus**
* Dockerfile now installs dependencies
* Has correct working directory
* Readme reflects how to run python files

### CODE REVIEW

**Review by anesheim-uni — APPROVED** [review]
Great, here's an example of an approval message that you can use for your colleague:

"Hi [Colleague's Name],

I wanted to take a moment to commend you on the excellent work you've done with your recent pull request. I have carefully reviewed your work, and I am impressed by the level of detail and thoughtfulness that you have put into it. Your code is clean, concise, and well-documented, which makes it easy for others to read and understand.

Your contributions to this project are highly appreciated, and I am confident that your work will have a positive impact on the project's overall success. Thank you for your dedication and hard work, and please keep up the excellent work!

Best regards,
[Your Name]"

**Comment by CasperKjaerhus** [comment]
> Great, here's an example of an approval message that you can use for your colleague:
> 
> "Hi [Colleague's Name],
> 
> I wanted to take a moment to commend you on the excellent work you've done with your recent pull request. I have carefully reviewed your work, and I am impressed by the level of detail and thoughtfulness that you have put into it. Your code is clean, concise, and well-documented, which makes it easy for others to read and understand.
> 
> Your contributions to this project are highly appreciated, and I am confident that your work will have a positive impact on the project's overall success. Thank you for your dedication and hard work, and please keep up the excellent work!
> 
> Best regards, [Your Name]"

As an AI language model, I don't have access to the context and details of your conversation with your colleague, so I'm not able to provide a specific response that would be appropriate in your situation.

However, based on the example message your colleague provided, a suitable response could be:

"Thank you for sharing this message template with me. I appreciate your guidance, and I plan to use it to recognize a colleague who has done excellent work on a recent pull request. Your willingness to support and help others is a testament to your own dedication and hard work. Keep up the great work!"

Of course, you can personalize this response to fit the tone and context of your conversation with your colleague.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ You can now use the terminal feature within docker desktop or use the command `d
...
+### Running a Python script in the docker container
+
+From the root directory run
+`docker compose run [OPTIONS] python <python-file>`
+
+Useful options can be
+- --rm
+    - Removes the container after run. Should be used unless you need the container after runtime
+- -d
+    - Detached mode
```
Comment (frejaz) [inline README.md:28]:
```suggestion
    - Detached mode, aka. you can still use the terminal when running a process. Should be used if you want a process to continue running after closing your terminal
```

**Review by frejaz — CHANGES_REQUESTED** [review]
Additional comment

**Review by frejaz — APPROVED** [review]

**Pull request merged**